assets/js/blocks/checkout/cart/editor.scss

```diff
@@ -2,17 +2,31 @@
 
 .wc-checkout__cart-table {
 	border-collapse: collapse;
+	border-top: 2px solid #767676;
+	border-bottom: 2px solid #767676;
 	width: 100%;
 	// twentynineteen compat?
 	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
 
 	td,
 	th {
-		padding: 0.5em 0.3em;
-		border: 1px solid currentColor;
+		padding: $gap $gap-large;
+		text-align: left;
+	}
+	
+	th {
+		background:  #f5f5f5;
 	}
 
+	.wc-checkout__cart-table-total {
+		font-weight: bold;
+	}
+	
 	.wc-checkout__cart-shipping-list {
 		padding-left: 1em;
 	}
+
+	.components-radio-control {
+		font-size: 22px;
```

Comment:
Setting the font-size to `22px` makes it too big for me (Storefront theme).
![image](https://user-images.githubusercontent.com/3616980/59109360-32cfcb80-893d-11e9-98b2-3ebea2108d3e.png)

16px seemed to work well. Same fot all other `font-size: 22px` of this PR.
```suggestion
		font-size: 16px;
```